war/src/main/java/io/lumeer/engine/controller/ConfigurationFacade.java

```diff
@@ -103,11 +106,11 @@ public Boolean getDataStorageUseSsl() {
     * @return Optional Integer value of the given key
     */
    public Optional<Integer> getConfigurationInteger(final String key) {
-      Optional<String> value = getConfigurationString(key);
-      if (value.isPresent()) {
-         return Optional.of(Integer.parseInt(value.get()));
+      try {
+         return getConfigurationString(key).map(Integer::parseInt);
+      } catch (NumberFormatException nfe) {
+         return Optional.empty();
```

Comment:
Is this really a good idea? Do not we want to at least trace log in such situations?